.github/workflows/test.yml

```diff
@@ -22,13 +22,12 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          # the current production setup
           - java: '8'
-            mongo: 'mongodb-linux-x86_64-3.6.13'
-            gradle_test: 'test'
+            mongo: 'mongodb-linux-x86_64-ubuntu2204-7.0.4'
+            wired_tiger: 'false'
```

Comment:
Wired tiger is the only option for non-enterprise MongoDB 4.2+

Reply:
Are you saying wired_tiger should always be `true` for non-enterprise MongoDB 4.2+

Reply:
I'm saying it doesn't matter as it's the only choice. If both options are 4.2+ we can just ignore it

Reply:
👍